lib/resources/example_json/r4_examples_charge_item.rb

```diff
@@ -513,5 +513,221 @@ module Resources
+                }
+              }
+            ],
+            'reason': [
+              {
+                'extension': [
+                  {
+                    'valueUnsignedInt': 1,
+                    'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/priority'
+                  }
+                ],
+                'coding': [
+                  {
+                    'extension': [
+                      {
+                        'valueCoding': {
+                          'system': 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/14002',
+                          'code': '3693'
+                        },
+                        'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/bill-code-schedule'
+                      }
+                    ],
+                    'system': 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/CodeSystem/BillCodes-ICD',
+                    'code': 'Y22'
+                  }
+                ]
+              }
+            ],
+            'extension': [
+              {
+                'extension': [
+                  {
+                    'valueCodeableConcept': {
+                      'coding': [
+                        {
+                          'extension': [
+                            {
+                              'valueCoding': {
+                                'system': 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/14002',
+                                'code': '3692'
+                              },
+                              'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/bill-code-schedule'
+                            }
+                          ],
+                          'system': 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/CodeSystem/BillCodes-MODIFIER',
+                          'code': '26'
+                        }
+                      ]
+                    },
+                    'url': 'code'
+                  },
+                  {
+                    'valueUnsignedInt': 1,
+                    'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/priority'
+                  }
+                ],
+                'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/modifier-code'
+              },
+              {
+                'valueReference': {
+                  'reference': 'Location/683731',
+                  'display': '1 West SA2'
+                },
+                'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/performing-location'
+              },
+              {
+                'extension': [
+                  {
+                    'id': '68985657',
+                    'valueString': 'Family Planning Indicator',
+                    'url': 'custom-attribute-name'
+                  },
+                  {
+                    'valueInteger': 689_856_57,
```

Comment:
Is this a valid Interger value with the _ characters in it?

Reply:
I believe it's a valid integer as the underscore is provided to improve readability @SantanuBanerjee your opinions.

Reply:
I think ruby lets us use the underscore (_) character to make large numbers more readable , mainly to separate thousands, but then we may want to decide if at all we want to use this ruby specific syntax in the json body.

Reply:
Replaced the value with less than thousand #diff-5cd46ad5f0af8a24217f67b656e77ce3624d6cefec3a18eddf7adc12bd47d35fR639